project/api/api_tests.py

```diff
@@ -0,0 +1,160 @@
+import datetime
+import json
+
+from django.test import TestCase, Client
+from django.db import IntegrityError
+
+from graphene.test import Client
+from graphql_relay import from_global_id
+
+from .schema import schema
+
+from .models import User, ClassProfile, Quiz, Question, Answer
+from .models import QuizResult, Response, QuizSession
```

Comment:
We can't import QuizResult anymore

Reply:
correct. fixes and changes to the tests are coming in my next pr